Automattic/jetpack
# Travis CI: Explicitly mention multisite running on master WP

#### Changes proposed in this Pull Request:

* Explicitly mention that we running both single and multisite tests on WP master


#### Testing instructions:






* Check the travis, make sure nothing breaks
* Can you say which build is running multisite tests?
* Check the build log. Can you easily spot the build info?

#### Proposed changelog entry for your changes:

* N/a
